src/server.rs

```diff
@@ -280,22 +280,40 @@ impl Server {
         function: FunctionInfo,
         is_const: bool,
     ) {
+
+        self.save_route(py, route_type, route, &function, is_const);
+
+        if &route[route.len() - 1..] == "/" {
+            self.save_route(py, route_type, &route[0..route.len() - 1], &function, is_const);
+        } else {
+            self.save_route(py, route_type, &format!("{}/", route), &function, is_const);
+        }
+    }
```

Comment:
@VishnuSanal , two suggestions
- https://doc.rust-lang.org/std/primitive.str.html#method.ends_with use this instead of route.len() - 1 and
- We should do this if check before the `self.save_route`

Reply:
> We should do this if check before the self.save_route

why? 🤔

Reply:
nvm. I have phrased it in a better way through code 😅